src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -488,7 +488,7 @@ public void AddClaims(IEnumerable<Claim> claims)
             }
         }
 
-        internal static object GetClaimValueUsingValueType(Claim claim)
+        private static object GetClaimValueUsingValueType(Claim claim)
```

Comment:
As I said above I changed to private since I was unable to find any references to these methods outside JwtPayload class. Coupled with the fact the assembly doesn't share its internals with any other assembly I can't see how it is a breaking change for anyone. 
Also if someone would be accessing this method via reflection calls, `BindingFlags` only have "Public" and "NonPublic" flags, i.e. there is no difference between "internal" and "private" modifiers as far as they go. 

I am happy to be corrected or proven wrong.